src/ontology/fbbt-edit.obo

```diff
@@ -122569,6 +122585,21 @@ relationship: part_of FBbt:00110893 ! adult midgut region R2
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-06-07T11:33:30Z
 
+[Term]
+id: FBbt:00058222
+name: sugar-sensing neuron of the adult protocerebrum
+def: "Chemosensory neuron in the posterior superior lateral protocerebrum that senses sugars in the hemolymph to regulate feeding. There are 4 of these cells per brain hemisphere. They express the fructose receptor Gr43a and the sugar receptor Gr64a, and secrete corazonin (Miyamato and Amrein, 2014; Fuji et al., 2015)." [FlyBase:FBrf0224445, FlyBase:FBrf0227698]
+subset: cur
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0041243 ! Gr43a
+intersection_of: has_soma_location FBbt:00007145 ! adult protocerebrum
+relationship: capable_of GO:0001582 ! detection of chemical stimulus involved in sensory perception of sweet taste
+relationship: capable_of GO:0030072 ! peptide hormone secretion
```

Comment:
you can put the more specific soma location relationship as a SubClassOf annotation:
'has soma location' some 'cell body rind of adult posterior superior lateral protocerebrum'

based on your definition, you can also add:
'capable of part of' some 'adult feeding behavior'
(or 'capable of' some 'regulation of feeding behavior', but I'm thinking that it may be better to move away from using 'regulation' terms and annotate just with behaviors so all neurons involved in feeding would be annotated with the same term)